src/utils/sidebarConfig.ts

```diff
@@ -96,6 +96,11 @@ async function getInventorySidebar(): Promise<SidebarRoot[]> {
           name: 'stock-balance',
           route: '/report/StockBalance',
         },
+        {
+          label: t`Batch`,
```

Comment:
Why is Batch in the sidebar? It may be searchable but like Location, it should not be placed in the sidebar.

Reply:
corrected